book/src/csi-driver-object.md

```diff
@@ -26,6 +26,9 @@ metadata:
 spec:
   attachRequired: true
   podInfoOnMount: true
+  volumeLifecycleModes:
```

Comment:
Add a comment here that it's added in 1.16

Reply:
Done.